package.json

```diff
@@ -159,7 +159,7 @@
     "babel-preset-react-native": "^1.9.0",
     "body-parser": "^1.18.2",
     "detox": "^7.4.3",
-    "edge-plugin-simplex": "https://github.com/Airbitz/edge-plugin-simplex.git#c1b81e1",
+    "edge-plugin-simplex": "https://github.com/Airbitz/edge-plugin-simplex.git",
```

Comment:
I would have left the commit hash in there, like `#a0df994290`, but this is fine. We can fix this next time we update the plugin.